commands/sandbox_util.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/digitalocean/doctl/do"
+)
+
+const (
+	// Minimum required version of the sandbox plugin code.  The first part is
+	// the version of the incorporated Nimbella CLI and the second part is the
+	// version of the bridge code in the sandbox plugin repository.
+	minSandboxVersion = "4.1.0-1.3.0"
+
+	// The version of nodejs to download alongsize the plugin download.
+	nodeVersion = "v16.13.0"
+
+	// noCapture is the string constant recognized by the plugin.  It suppresses output
+	// capture when in the initial (command) position.
+	noCapture = "nocapture"
+
+	// credsDir is the directory under the sandbox where all credentials are stored.
+	// It in turn has a subdirectory for each access token employed (formed as a prefix of the token).
+	credsDir = "creds"
```

Comment:
Doesn't look like the `credsDir` const from the commands package is being used either.

Reply:
Yes.   Fixed.